src/test/java/teammates/test/cases/action/InstructorHomePageActionTest.java

```diff
@@ -78,7 +79,7 @@ public void testExecuteAndPostProcess() throws Exception {
         a = getAction(addUserIdToParams(instructorWithMultipleCourses, submissionParams));
         r = getShowPageResult(a);
 
-        assertEquals("/jsp/instructorHome.jsp?error=false&user=" + instructorWithMultipleCourses,
+        assertEquals(getPageResultDestination("/jsp/instructorHome.jsp", false, instructorWithMultipleCourses),
```

Comment:
"/jsp/instructorHome.jsp" can be replaced with `Const.ViewURIs.INSTRUCTOR_HOME`.